euth_wagtail/static/scss/components/forms/_general-form.scss

```diff
@@ -87,6 +87,19 @@
     }
 }
 
+.form-check__label {
```

Comment:
Should this and inputs remain in general-form or be placed in a new generic-form or form file?